CMakeLists.txt

```diff
@@ -83,6 +83,7 @@ set (OPENVDB_ROOT_DIR "" CACHE STRING "Custom OpenVDB path")
 option (USE_TBB "Use Intel TBB if found" ON)
 set (TBB_ROOT_DIR "" CACHE STRING "Custom Intel TBB path")
 option (USE_FFMPEG "Use FFmpeg if found" ON)
+option (USE_REDSDK "Use RedSDK if found" ON)
```

Comment:
Maybe let's set this to OFF as the default, just so we can't possibly break anybody not trying to build for Red, until we have made more progress and tested more thoroughly.

Reply:
Done!